scripts/facts.runs/facts_runs.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
+# Stop script when non-facts error occurs
+set -e
```

Comment:
Maybe some sort of echo message, like "An error occurred. Exiting the script" or something along those lines @JMGilbert